### PR TITLE
Fix link to client hint docs on HTTP working group

### PR DESCRIPTION
**Pull request by hamishwillee**
The link client hint docs on "website of the HTTP working group" was broken. This fixes the link to point to latest draft doc.

### CODE REVIEW

File: files/en-us/web/http/headers/index.html [hunk 1]
```diff
@@ -76,7 +76,7 @@ <h2 id="Caching">Caching</h2>
 
 <h2 id="Client_hints">Client hints</h2>
 
-<p>HTTP {{Glossary("Client_hints", "Client hints")}} are a work in progress. Actual documentation can be found on the <a href="https://httpwg.org/http-extensions/client-hints.html">website of the HTTP working group</a>.</p>
+<p>HTTP {{Glossary("Client_hints", "Client hints")}} are a work in progress. Actual documentation can be found on the <a href="https://httpwg.org/http-extensions/draft-ietf-httpbis-client-hints.html">website of the HTTP working group</a>.</p>
```
Comment (hamishwillee) [inline files/en-us/web/http/headers/index.html:79]: Actually, perhaps https://tools.ietf.org/html/draft-grigorik-http-client-hints-03 is better?

Comment (Elchi3) [reply files/en-us/web/http/headers/index.html:79]: nah,I think the one have there is the most current. -03 looks outdated.

**Review by Elchi3 — APPROVED** [review]

**Comment by Elchi3** [comment]
hm, I was trying to resolve the conflict but it looks like there is a much better description now for Client Hints. Is this PR needed at all?

**Comment by hamishwillee** [comment]
> hm, I was trying to resolve the conflict but it looks like there is a much better description now for Client Hints. Is this PR needed at all?

@Elchi3 On one hand no, because the spec is about the rationale behind client hints and says very little about what the available hints actually are. On the other hand yes, because we should always point to the spec.

But how about we change this to something like "specification information is here ..."rather than "actual documentation is here"?

**Comment by Elchi3** [comment]
Agree, the spec could be linked to for more info for now.

And I've nudged Chris to actually discuss changes in PRs so we can get on the same page here :)

**Comment by hamishwillee** [comment]
Great. So I have appended the current spec link to the end of @chrisdavidmills text.
"(the specification can be found on the <a href="https://httpwg.org/http-extensions/draft-ietf-httpbis-client-hints.html">HTTP working group website</a>)."

**Review by Elchi3 — APPROVED** [review]
Okay cool 👍

**Pull request merged**